tests/wpt/meta/webxr/__dir__.ini

```diff
@@ -0,0 +1 @@
+prefs: [dom.webxr.hands.enabled: true]
```

Comment:
Should we enable this pref by default now? Originally this was introduced because the hands spec was still very new and being iterated on rapidly.

Reply:
We probably can yeah, the hands spec has pretty solid for a while now